src/CMakeLists.txt

```diff
@@ -5,9 +5,9 @@ cmake_minimum_required(VERSION 3.2)
 
 project(osconfig)
 
-set(MAJOR_VERSION "0" CACHE STRING "Major version to use")
-set(MINOR_VERSION "4" CACHE STRING "Minor version to use")
-set(PATCH_VERSION "1" CACHE STRING "Patch version to use")
+set(MAJOR_VERSION "1" CACHE STRING "Major version to use")

```

Comment:
>"Major version to use") [](http://example.com/codeflow?start=35&length=23)

Can we remove "to use"? (which is obvious)

And have just "Major/Minor/Patch version"?

Reply:
I agree 👍🏻